templates/launchd.plist

```diff
@@ -73,17 +75,17 @@
     <integer>30</integer>
 
     <key>StandardOutPath</key>
-    <string>/usr/local/var/log/buildkite-agent.log</string>
+    <string>/Users/your-build-user/.buildkite/log/buildkite-agent.log</string>
 
     <key>StandardErrorPath</key>
-    <string>/usr/local/var/log/buildkite-agent.log</string>
+    <string>/Users/your-build-user/.buildkite/log/buildkite-agent.log</string>
 
     <key>EnvironmentVariables</key>
     <dict>
       <key>PATH</key>
       <string>/usr/bin:/bin:/usr/sbin:/sbin:/usr/local/bin</string>
 
-      <key>BUILDKITE_AGENT_TOKEN</key>
```

Comment:
I actually think you can get rid of this entire ENV all together :) You don't need it since you've got https://github.com/buildkite/agent/pull/143/files#diff-d2f5f813d0c1aed057afb6b8b014be21R56